### ISSUE TITLE
Error: Tensor Size Mismatch in Florence2Run on Mac M2

### ISSUE THREAD

**Issue by vglushkov** [kijai/ComfyUI-Florence2#64]
When setting num_beams to 1, the node runs fine. With any other setting it returns the error.

`The expanded size of the tensor (1025) must match the existing size (14) at non-singleton dimension 8. Target sizes: [1025]. Tensor sizes: [14]`

<img width="1226" alt="Florence2Run_error" src="https://github.com/user-attachments/assets/1d10b390-c3f5-4ba2-9b4a-0afb6dc118ce">

**Comment by 21Machines** [comment]
I can't get it to run at all, Mac Studio 96GB 38-core GPU.  Are you running this command at startup;  PYTORCH_ENABLE_MPS_FALLBACK=1

**Comment by 21Machines** [comment]
Yes, I notice that after I posted my comment.  Just about every ComfyUI workflow comes back with an error, "not supported on MPS".    Guess I'll need to buy a CUDA machine if I want to explore AI.  Shame.
